workflows/epigenetics/ChIPseq_PE/ChIPseq_PE-tests.yml

```diff
@@ -0,0 +1,106 @@
+- doc: Test outline for ChIPseq_PE
+  job:
+    PE fastq input:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: Collection
+        type: paired
+        identifier: wt_H3K4me3
+        elements:
+        - identifier: forward
+          class: File
+          location: https://zenodo.org/record/1324070/files/wt_H3K4me3_read1.fastq.gz
+          filetype: fastqsanger
+        - identifier: reverse
+          class: File
+          location: https://zenodo.org/record/1324070/files/wt_H3K4me3_read2.fastq.gz
+          filetype: fastqsanger
+    adapter_forward: 'GATCGGAAGAGCACACGTCTGAACTCCAGTCAC'
+    adapter_reverse: 'GATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT'
+    reference_genome: 'mm10'
+    effective_genome_size: 1870000000
+  outputs:
+    MultiQC webpage:
+      asserts:
+        - that: "has_text"
+          text: "wt_H3K4me3"
+        - that: "has_text"
+          text: "<a href="#cutadapt_filtered_reads" class="nav-l2">Filtered Reads</a>"
+        - that: "has_text"
+          text: "<a href="#bowtie2" class="nav-l1">Bowtie 2 / HiSAT2</a>"
```

Comment:
```suggestion
          text: "<a href=\"#bowtie2\" class=\"nav-l1\">Bowtie 2 / HiSAT2</a>"
```

Reply:
You can just skip the outer quotes, they're not necessary:
```
In [8]: yaml.safe_load("""
   ...:         - that: "has_text"
   ...:           text: <a href="#bowtie2" class="nav-l1">Bowtie 2 / HiSAT2</a>
   ...: """)
Out[8]:
[{'that': 'has_text',
  'text': '<a href="#bowtie2" class="nav-l1">Bowtie 2 / HiSAT2</a>'}]
```

Reply:
thanks